### ISSUE TITLE
Add directions to use patcher standalone to readme

### ISSUE THREAD

**Issue by JeffGainsNGames** [randovania/open-prime-rando#133]

**Comment by duncathan** [comment]
also worth adding documentation about development - particularly about how to set up a testing environment and updating hashes

**Comment by henriquegemignani** [comment]
Maybe we could have the updating hashes step be a command you can just run instead of a line of code you comment out. 

And I don't know why only _now_ I've realized it could have just been a command line argument for the tests this entire time. Ooops.

**Comment by duncathan** [comment]
would be great if we found a way to not require contributors to have PAL, too. maybe a way for maintainers to leave a comment on a PR, which triggers a bot on our self-hosted runner to update the hashes and commit to the branch?

**Comment by henriquegemignani** [comment]
Yes, but that's high effort. And gotta be careful to not accidentally allow randoms to run arbitrary code on the self-hosted runner.

**Comment by duncathan** [comment]
a quick and dirty guide that I wrote for Reggie. good place to start for anyone writing real documentation

> for any development work, you first need to clone the repo and then set up the environment. clone the repo however you like, and then run these commands in your terminal:
`uv sync --all-groups --all-extras` (if you don't have uv installed you'll need to install it first, but most RDV contributors should already have it)
`pre-commit install` (same thing, you might need to install pre-commit first but RDV contributors should already have it) 

> to run the tests, you need to have two environment variables set. setting them depends on your OS, so look up how to do it on your particular system. you need to set `PRIME2_ISO` to the path to your vanilla NTSC ISO, and `PRIME2_PAL_ISO` to the path to your vanilla PAL ISO.

> to run the tests, use `uv run -m pytest -n auto`. this runs every test we have. if the tests in `test_patcher.py` fail because of mismatched hashes, it's because our saved hashes don't match the output with your changes. more explanation here https://discord.com/channels/914291389293027329/914294505107251231/1507046415464141040

> to update the hashes, if you're intentionally changing the contents of a file, go to `test_patcher.py` and uncomment the `_update_hashes_file()` lines in `test_ntsc_export()` and `test_pal_export()`. then run `uv run -m pytest tests\echoes\test_patcher.py`. the tests will fail, and when they fail they will update the saved hashes file. then, recomment the lines

> once you've done this, if you'd like, you can run `uv run -m pytest -n auto` again to make sure that everything is passing before you push your code
